### ISSUE TITLE
Failed setting permission exec for all

### ISSUE THREAD

**Issue by cup** [luarocks/luarocks#974]
Previously LuaRocks was giving me problems because `TMP` was not set and no
fallback is offered currently with Windows.

However even setting `TMP` only reveals another problem:

~~~
$ luarocks install Lua-cURL
Installing https://luarocks.org/lua-curl-0.3.9-1.src.rock

Error: Could not fetch rock file: Failed creating temporary directory
luarocks-rock-lua-curl-0.3.9-1: Failed setting permission exec for all
~~~

**Comment by cup** [comment]
https://github.com/luarocks/luarocks/issues/961

**Comment by daurnimator** [comment]
Was this fixed in master already with https://github.com/luarocks/luarocks/commit/914a2b7503b15c40894ac9efb8f33352f1a75cd1 ?

**Comment by cup** [comment]
@daurnimator yes i think so - I am using Cygwin so doing a build from here

https://github.com/luarocks/luarocks/wiki/Installation-instructions-for-Unix

allows to install packages

**Comment by cup** [comment]
I am going to close this - appears to be resolved as of v3.1.0 and later

**Comment by neunmalelf** [comment]
version 3.7.0 same error message
